src/Analyzers/Core/Analyzers/UseAutoProperty/AbstractUseAutoPropertyAnalyzer.cs

```diff
@@ -322,19 +274,11 @@ private void Process(AnalysisResult result, SemanticModelAnalysisContext context
                 propertyDeclaration.GetLocation(),
                 variableDeclarator.GetLocation());
 
-            var option = context.GetAnalyzerOptions().PreferAutoProperties;
-            if (option.Notification.Severity == ReportDiagnostic.Suppress)
-            {
-                // Avoid reporting diagnostics when the feature is disabled. This primarily avoids reporting the hidden
-                // helper diagnostic which is not otherwise influenced by the severity settings.
-                return;
-            }
```

Comment:
this check moved up higher, to where we actually first look at the prefer-auto-prop option.  no point in getting all this way just to bail out now.  might as well bail out initially.